output/plugins/forms/xml/forms.cppcode

```diff
@@ -407,7 +407,26 @@ Written by
 			#ifequal $aui_managed "1"
 			@{ @#include &lt;wx/aui/aui.h&gt; @}
 		</template>
-		<template name="settings">
+        <template name="code_init">
+            #ifnotnull $create_children
+            @{
+            #ifequal $create_children "0"
+            @{ { #nl
+            } #nl #nl
+            void $name::Init()
+            @}
+            @}
```

Comment:
This is hard to read because of no indentation

Reply:
That will be fixed, I was just trying stuff out to see if it would work.

Reply:
Latest push should have addressed the issues you raised (thanks).